ipvs/netlink.go

```diff
@@ -424,6 +424,9 @@ func assembleDestination(attrs []syscall.NetlinkRouteAttr) (*Destination, error)
 		switch attrType {
 		case ipvsDestAttrAddress:
 			ip, err := parseIP(attr.Value, syscall.AF_INET)
+			if len(attr.Value) > 4 && err != nil {
```

Comment:
Can you change it like below. Always better to check err first and then go with second validation. 
if err != nil  && len(attr.Value) >  {